e2e/jest.config.js

```
@@ -3,7 +3,7 @@ const { TEST_MODE } = process.env;
 
 module.exports = {
   "verbose": true,
-  "bail": 3,  // Stop running tests after `n` failures
+  "bail": 1,  // Stop running tests after `n` failures
```

Fail fast in CI. CI will end after 1 test failure.